EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+title: Predictable Proof-of-Work (POW) Sunsetting 
+author: @Query0x
+discussions-to: <tbd>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-13
+---
+
+## Simple Summary
+Provides for a predictable, issuance neutral reduction of block rewards to 1 over a one year time span while reducing the uncertainties and risks associated with sudden changes to mining economics.
+
+## Abstract
+Sets the block reward to 3 ETH and then incrementally decreases it every block for 2,362,000 blocks (approximately 1 year) until it reaches 1 ETH.
+
+## Motivation
+Unnecessarily abrupt changes to the Ethereum ecosystem cause disruption and disharmony resulting in the disenfranchisement of community members while undermining stability and confidence.  While moves from Proof-of-Work to Proof-of-Stake will undoubtedly cause friction between those community members vested in either, all benefit from a measured, predictable transition.
+
+This proposal:
+
+1) Is issuance neutral.  There is no overall increase in mining rewards.
```

Comment:
This is not true.  It is issuance neutral over 1 year, it is net negative issuance beyond that (relative to current).  Also, EIPs should try to be worded in a way that avoids comparing it to the current state of things and instead should focus on addressing why this way is a good way on its own (regardless of current system).  Sometimes this isn't possible, but in this case I think this line isn't adding technical merit as it is.

Reply:
It is issuance neutral over the proposed timeframe it is in effect.  I think it is important to be unambiguous about that point given community comments to eip-3368.

Reply:
```suggestion
1) Is issuance neutral over 1 year, and reduces issuance beyond that.
```
The current text is incorrect without bounding on the time frame.  This proposal *decreases* issuance over any time span longer than 1 year.

Reply:
> given community comments to eip-3368.
Keep in mind, EIPs are technical documents, they are not an appropriate place for arguments.  The motivation section's only purpose is to give people a high level overview of why this change is desirable, it is not meant to be an exhaustive argument in favor of a particular EIP nor counter every point of opposition heard in the wild.

It does need to be *technically correct* though, which is why I'm pressing on changing the current wording (which is technically incorrect).